deploy/prepare_deployment.bat

```diff
@@ -4,6 +4,11 @@ for /f "tokens=2-4 delims=/ "  %%a in ("%date%") do (set MM=%%a& set DD=%%b& set
 set build_date=%YYYY%-%MM%-%DD%
 set git_rev=%APPVEYOR_REPO_COMMIT:~0,8%
 
+rem Default values because %bintray_deploy% is currently unused
+set pkg_name=master
+set git_rev=%APPVEYOR_REPO_COMMIT:~0,8%
```

Comment:
This is declared above already but since we have to change this section again when introducing `deploy_to_bintray.bat` this shouldn't be a show stopper.